### PR TITLE
Add documentation for Conventions Component

### PR DESCRIPTION
**Pull request by rszumlakowski**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

TAP 1.8 only

### CODE REVIEW

File: supply-chain/catalog/conventions.hbs.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Conventions Component
+
+## Description
+
+This document describes the Conventions Component for Tanzu Supply Chains.  The Conventions Component is a way to invoke
+the Conventions Controller in a Tanzu Supply Chain.  Please see the [Conventions Controller](../../cartographer-conventions/about.hbs.md)
+documentation itself for more details on the Conventions Controller service itself.
+
+## API
```
Comment (rashedkvm) [inline supply-chain/catalog/conventions.hbs.md:9]: **Suggestion:** Can we add the API Schema here?

Comment (rszumlakowski) [reply supply-chain/catalog/conventions.hbs.md:9]: Which API Schema do you mean?  The YAML from the Component definition?

Comment (squeedee) [reply supply-chain/catalog/conventions.hbs.md:9]: I think the default config would be useful

`spec.env` is an array of name, value pairs....

**Review by squeedee — APPROVED** [review]
Could add more detail, but happy for it to be a later merge.

File: supply-chain/catalog/conventions.hbs.md (hunk 1 above)
Comment (rszumlakowski) [reply supply-chain/catalog/conventions.hbs.md:9]: Fair enough.  I explained it in the prose below but I will add something more concise in the API section, too.

**Review by AMGallo7 — APPROVED** [review]

**Review by squeedee — APPROVED** [review]

**Pull request merged**